### ISSUE TITLE
Cite buttons do not have unique IDs

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/happy-heron#3188]
The "Cite" button is repeated multiple times; form field labels should be unique. 

originally in issues:
- #3176
- #3172

from SODA report:

[Issue]
There are form field labels that are not unique.  The "Cite" button is repeated multiple times.

[User Impact]
Form fields that share labels (visual and programmatic), including input fields, select fields, buttons, checkboxes, radio buttons, and textareas, can be difficult to differentiate from one another for multiple types of users with disabilities.

the offending spot:

![image](https://github.com/sul-dlss/happy-heron/assets/96775/11ed9044-b961-489f-b953-cfdef06f56a8)

[Recommendation]
Provide unique visual and explicit labels (when present) for all form elements on the page. When the visual text of the element cannot be changed, provide a title attribute or use aria-label or aria-labelledby to create a unique label.

Form fields, including buttons, that are contained within a fieldset grouping can have non-unique names as long as the fieldset legend text distinguishes the label from fields with the same name that are not in the fieldset.

When using aria-label or a title attribute, consider placing the on-screen label as the beginning of these attributes to ensure direct voice access by users of speech recognition software. Users of speech recognition software may voice the visual label of the form field to interact with the field.

**Comment by ndushay** [comment]
@astridu @amyehodge we got an accessibility ding because the Cite button is repeated multiple times (and it's a form field).  I could change it to a link??

![image](https://github.com/sul-dlss/happy-heron/assets/96775/11ed9044-b961-489f-b953-cfdef06f56a8)

**Comment by astridu** [comment]
Yes, please. And the PURL will get read before it and the copy link so I don't think we need to specify what it is copying/citing.

**Comment by aaron-collier** [comment]
@ndushay / @astridu oddly enough, when I change the `cite` buttons to links, it removes them from the tab order and they are not tabbable, I'm looking into that, but it may not be the right direction here.
